crypto/dsa/eddsa/eddsa.go

```diff
@@ -0,0 +1,76 @@
+// Package eddsa implements the EdDSA signature schemes as per RFC 8032
+// https://tools.ietf.org/html/rfc8032. Note: Currently only Ed25519 is supported
+package eddsa
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/tbd54566975/web5-go/jwk"
+)
+
+const (
+	JWA     string = "EdDSA"
+	KeyType string = "OKP"
+)
+
+var AlgorithmIDs = map[string]bool{
```

Comment:
As above, should be private.

Reply:
done https://github.com/TBD54566975/web5-go/commit/2d4f7fe99202270e1714df499bb596fc1413525b